### ISSUE TITLE
Wrong error_1.csv with multiple imports

### ISSUE THREAD

**Issue by ebogaard** [salesagility/SuiteCRM#4116]
#### Issue

When importing several files, it's sometimes nice to save the log with errors and/or records with errors. This file should always show the information from the last import. This isn't the case in 7.8.x.

#### Expected Behavior

For every import, a new list with the errors/records of the last import should be available.

#### Actual Behavior

When downloading, for example, the list with errors after the first import, all is well.
After a second or any other following import, you get the same list you've download after the first import. The summary on screen shows the correct information, though.
When looking on the file system, I can see the expected files in upload/import/: these files contain the correct information from the last import.

It doesn't matter if the following import is in the same module or another module: the described behaviour always happens with any following import.

#### Steps to Reproduce

1. Import some records with errors in any module.
2. Download the list with errors
3. Import some records with different errors a second time.
4. Download the list with errors: you get the same list as in step 2, instead of a list with new errors.

#### Your Environment

* SuiteCRM Version used: 7.8.1
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): Firefox 55
* Environment name and version (e.g. MySQL, PHP 7): MariaDB 10.1, PHP 5.6
* Operating System and version (e.g Ubuntu 16.04): Centos 6

**Comment by TeuZz** [comment]
Hey, I got the same problem last week with SuiteCRM 7.11.
The problem is the cache. When I download my list, browser seems cache it.

I found only one solution for now, add a header in download.php file like this :
`        header('Cache-Control: no-cache, no-store, must-revalidate, proxy-revalidate');`

Maybe anyone knows why this file is cache and if is very important to cache it ?
If it's not important, I'll be able to purpose a pull resquest with the new 'header' instruction.
